pkg/kubectl/cmd/run.go

```diff
@@ -117,9 +117,11 @@ func addRunFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("expose", false, "If true, a public, external service is created for the container(s) which are run")
 	cmd.Flags().String("service-generator", "service/v2", "The name of the generator to use for creating a service.  Only used if --expose is true")
 	cmd.Flags().String("service-overrides", "", "An inline JSON override for the generated service object. If this is non-empty, it is used to override the generated object. Requires that the object supply a valid apiVersion field.  Only used if --expose is true.")
+	cmd.Flags().Bool("silent", false, "If true, hide the 'waiting for pod to be running' message")
```

Comment:
Did you mean we make it a global flag? 

What'd other messages to be hidden? Something like when `kubectl delete pod/foo --silent` we don't print `pod "foo" deleted` but only return error messages?